pandas/io/common.py

```diff
@@ -144,21 +147,22 @@ def _stringify_path(filepath_or_buffer):
     strings, buffers, or anything else that's not even path-like.
     """
     if hasattr(filepath_or_buffer, "__fspath__"):
-        return filepath_or_buffer.__fspath__()
+        # https://github.com/python/mypy/issues/1424
```

Comment:
When we drop 3.5 support do you think we can just do `isinstance(filepath_or_buffer, os.PathLike) here instead?